docs/DeveloperGuide.md

```diff
@@ -20,18 +197,18 @@
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ... | I want to ...             | So that I can ...                                           |
+| ------- | -------- | ------------------------- | ----------------------------------------------------------- |
+| v1.0    | new user | see usage instructions    | refer to them when I forget how to use the application      |
+| v2.0    | user     | find a to-do item by name | locate a to-do without having to go through the entire list |
 
 ## Non-Functional Requirements
 
 {Give non-functional requirements}
 
 ## Glossary
 
-* *glossary item* - Definition
+- _glossary item_ - Definition
 
 ## Instructions for manual testing
```

Comment:
Perhaps you could elaborate more on how various testing is performed in your application, such as unit testing, integration testing, system testing, etc.